lib/ansible/module_utils/mysql.py

```diff
@@ -0,0 +1,56 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
module utils code does not use the GPL but a BSD license, this is so others can reuse this code when distributing independent 3rd party modules,